### DISCUSSION TITLE
《我獨自升級》已上線！

### DISCUSSION THREAD

**Discussion by vchien03** [1101-GS4719-C/Python-Community-GS4719-C#1884] [General]
當時在看《我獨》的漫畫就喜歡到一刷再刷，之後又看到要出動畫的消息，我已經期待很久了！！從去年關注到現在，不知道有沒有人跟我一樣呢😍 
終於！在1/7正式播出啦~除了動畫本身，OST也是一大看點！片頭曲<LEveL>竟然請來動畫配樂大師澤野弘之譜曲，與韓國人氣男團TXT跨界合作！
*可在MyVideo、巴哈姆特上觀看
*注意長時間使用電子產品影響視力

**Comment by 110101501** [comment]
我有在巴哈動畫瘋看！不過我是動畫黨的因此還沒有看過漫畫。在作品中成哥能力一直提升，根本就真的是在開外掛，超強……
順帶一提，官方前幾天也公布了要出第二季的消息，可以好好期待了一下！

**Comment by LinChloe** [comment]
我看了兩集後，因為更新太慢就去看漫畫了。
後面劇情還可以，有點爽文的感覺。
(我最喜歡的角色是蟻王，它的個性很可愛!!)
到最後還有男主的愛情發展歡迎大家去看看。

**Comment by KH1126** [comment]
我覺得我獨自升級很好看，在把第一集動畫看完後，我還有去看後續的漫畫。
之後轉職後的說的”站起來”太帥了。
我蠻推薦這部作品。

**Comment by yubo0oo** [comment]
我不推

**Comment by poiu6fi** [comment]
前面我也覺得滿好看的，可是到了後面就有點不知道在幹嘛
